greynoise/config.schema.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
+gn_api_key:
+    description: "API Key for authenticating to GreyNoise API"
```

Comment:
I would include some text about the default value will be pulled from the `gn_api_key` key in key-value store.

Reply:
Updated.